src/gisida.d.ts

```diff
@@ -1,27 +1,28 @@
 declare module 'gisida' {
+  import { Dictionary } from '@onaio/utils';
   import { Map as mbMap } from 'mapbox-gl';
   import { ParseResult } from 'papaparse';
   import { AnyAction } from 'redux';
 
   /** interface for generic object */
-  interface FlexObject {
+  interface Dictionary {
     [key: string]: any;
   }
```

Comment:
This looks like it should be removed, thoughts?

Reply:
yeah for sure missed that